package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.0.0",
   "private": true,
   "license": "MIT",
-  "scripts": {},
+  "scripts": {
+    "format": "prettier-eslint --config .prettierrc.json --write \"**/*.{jsx,js,ts,tsx}\""
```

Comment:
This will format every single `jsx, js, ts, tsx` within newm-web, should we isolate or should this be fine

Reply:
I was able to add the app/package specific commands to the `project.json` files, I also added the documentation to our readme to run format across any or all apps/packages.

- Single App or Package: `nx run <app-or-package>:format`
- All Apps and Packages: `nx run-many -t format`

I attempted but was unable to get anything configured in the Makefile. I'm unsure of what to actually include or how to implement the formatting through the Makefile. I did find that nx does have docs on implementing root level NPM scripts: https://nx.dev/recipes/running-tasks/root-level-scripts, if we want to do a format for the entire `newm-web`, but with the above format commands I don't think it's necessary. Let me know your thoughts.

Reply:
Got it, will roll back the README change